tasks/cat2.yml

```diff
@@ -3052,6 +3108,118 @@
   ignore_errors: yes
   tags: [ 'cat2' , 'V-38691' , 'bluetooth' ]
 
+- name: V-38619 Medium  There must be no .netrc files on the system
```

Comment:
These do not follow the style guide in https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md